oci/table_oci_apigateway_api.go

```diff
@@ -0,0 +1,268 @@
+				Description: "The OCID of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "display_name",
+				Description: "A user-friendly name. Does not have to be unique, and it's changeable.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The time this resource was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+
+			// other columns
+			{
+				Name:        "time_updated",
+				Description: "The OCID of dedicated VM host.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeUpdated.Time"),
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The current state of the API.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lifecycle_details",
+				Description: "A message describing the current lifecycleState",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "specification_type",
+				Description: "Type of API Specification file.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "validation_results",
+				Description: "Status of each feature available from the API.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// json fields
+
+			// tags
+			{
+				Name:        "defined_tags",
+				Description: "Defined tags for this resource. Each key is predefined and scoped to a namespace.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "freeform_tags",
+				Description: "Free-form tags for this resource. Each tag is a simple key-value pair with no predefined name, type, or namespace.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(apiTags),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Id").Transform(ociRegionName),
+			},
+			{
+				Name:        "compartment_id",
+				Description: ColumnDescriptionCompartment,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CompartmentId"),
+			},
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenant,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTenantId,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listApigatewayApis(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func listApiGatewayApis(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```
Please make changes where ever required.